.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+const isCI = require("is-ci");
+
+/** @type {import("prettier").Options} */
+module.exports = {
+    printWidth: 100,
```

Comment:
Why is this 100 actually? I think it's 120 in tslint

Reply:
In linters max length is used as a strict limit, but it's possible to make lines shorter. In prettier `printWidth` it more like average length, as prettier tries to as fit much as possible code into `printWidth`, so usually it's better to keep it less than it was with linter. See https://prettier.io/docs/en/options.html#print-width and https://prettier.io/docs/en/rationale.html#print-width for details.
Though I'm fine either way, the difference between 100 and 120 is quite small.